ci-operator/config/openshift/machine-api-provider-azure/openshift-machine-api-provider-azure-main.yaml

```diff
@@ -0,0 +1,90 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.9
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9
+  to: azure-machine-controllers
+promotion:
+  disabled: true
+  name: "4.10"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: goimports
+  commands: IS_CONTAINER=TRUE make goimports
+  container:
+    from: src
+- as: govet
+  commands: |
+    IS_CONTAINER=TRUE make vet
+  container:
+    from: src
+- as: generate
```

Comment:
Should we delete this generate job probably? this was moved to openshift/api, isn't it?